env/work/srw_python/srwl_bl.py

```diff
@@ -1413,56 +1448,195 @@ def srwl_uti_ext_options(_arOpt):
     """
     return srwl_uti_merge_options(srwl_uti_std_options(), _arOpt)
 
+
 #****************************************************************************
-def srwl_uti_parse_options(_descr):
```

Comment:
OK. This probably was already implemented earlier (from here to ln 1642), with some modifications (renaming functions), during earlier discussions (needs to be re-checked).